tests/Unit/Services/Docker/Local/ConfigExceptionTest.php

```diff
@@ -2,24 +2,27 @@
 
 namespace Tests\Unit\Services\Docker\Local;
 
-use Tests\TestCase;
-use RuntimeException;
 use App\Runners\CommandRunner;
-use phpmock\mockery\PHPMockery;
 use App\Services\Docker\Local\Config;
+use Illuminate\Config\Repository;
 use Illuminate\Support\Facades\Storage;
+use phpmock\mockery\PHPMockery;
+use RuntimeException;
+use Tests\TestCase;
 
-class ConfigExceptionTest extends TestCase {
+final class ConfigExceptionTest extends TestCase {
```

Comment:
Because we shouldn't be using inheritance in tests outside of having a base test that may set up a bunch of stuff all tests need.

In the event anyone else needs to write tests here, it should make it clear that they shouldn't be extending any existing tests when writing their own.

Reply:
That's what I thought, just wanted to make sure I'm on the same page.